app/kafka/manager/actor/cluster/KafkaStateActor.scala

```diff
@@ -295,7 +295,7 @@ case class KafkaManagedOffsetCache(clusterContext: ClusterContext
               lastUpdateTimeMillis = System.currentTimeMillis()
             } catch {
               case e: Exception =>
-                warn("Failed to process a message from offset topic!", e)
+                warn(s"Failed to process a message from offset topic on cluster ${clusterContext.config.name}!", e)
```

Comment:
@whithajess, in general, you have a point that if a variable is undefined in a catch, then there's no point in catching the first exception if we just generate another one anyway.  However, in this case `${clusterContext.config.name}` is already used on [line 242](https://github.com/Innometrics/kafka-manager/blob/dd92b03c6a0c791c197bb4cb0fa928f5c1b66b51/app/kafka/manager/actor/cluster/KafkaStateActor.scala#L242), and everything is wrapped in a try anyway.  This change wouldn't be executed if `${clusterContext.config.name}` wasn't defined.